### PR TITLE
Parser for command 'ovs-vsctl list bridge'

### PR DESCRIPTION
**Pull request by ppandey-redhat**
New parser for parsing command `ovs-vsctl list bridge`.

Test Coverage (100%):

    `insights/parsers/ovs_vsctl_list_bridge.py               31      0     16      0   100%`

### CODE REVIEW

File: insights/parsers/ovs_vsctl_list_bridge.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+from insights.core.filters import add_filter
+from insights.parsers import SkipException, optlist_to_dict
+from insights.specs import Specs
+
+FILTERS = ["name", "other_config", "mac-table-size"]
+add_filter(Specs.ovs_vsctl_list_bridge, FILTERS)
+
+
+@parser(Specs.ovs_vsctl_list_bridge)
+class OVSvsctlListBridge(LegacyItemAccess, CommandParser):
```
[the remark this answers is not in the document]
Comment (ppandey-redhat) [reply insights/parsers/ovs_vsctl_list_bridge.py]: @xiangce - Acknowledged. Changes made in the latest commit.

File: insights/parsers/ovs_vsctl_list_bridge.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+       data types as string, numbers, list or dictionary.
+       The class provides attribute ``data`` as list with lines parsed
+       line by line based on keys for each bridge.
+
+       Attributes:
+           data (list): A list containing dictionary elements where each
+                        element contains the details of a bridge.
+
+       Raises:
+        SkipException: When file is empty or value is not present for a key.
```
[the remark this answers is not in the document]
Comment (ppandey-redhat) [reply insights/parsers/ovs_vsctl_list_bridge.py]: @xiangce - Acknowledged. Changes made in the latest commit.

File: insights/parsers/ovs_vsctl_list_bridge.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    True
+"""
+
+import re
+from insights import LegacyItemAccess, CommandParser, parser
+from insights.core.filters import add_filter
+from insights.parsers import SkipException, optlist_to_dict
+from insights.specs import Specs
+
+FILTERS = ["name", "other_config", "mac-table-size"]
```
[the remark this answers is not in the document]
Comment (ppandey-redhat) [reply insights/parsers/ovs_vsctl_list_bridge.py]: @xiangce - Acknowledged. Changes made in the latest commit.

File: insights/parsers/ovs_vsctl_list_bridge.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        """
+        # No content found or file is empty
+        if len(content) == 0:
+            raise SkipException("Empty file")
+
+        self.data = []
+        content = ",".join(content).strip()
+
+        # Split the content on the basis of key 'name' and extract bridge details
+        key_list = [key.start() for key in re.finditer("name", content)]
```
[the remark this answers is not in the document]
Comment (ppandey-redhat) [reply insights/parsers/ovs_vsctl_list_bridge.py]: @xiangce - Acknowledged. Changes made in the latest commit.

File: insights/specs/default.py [hunk 5]
```diff
@@ -548,6 +548,7 @@ def semid(broker):
     ovirt_engine_ui_log = simple_file("/var/log/ovirt-engine/ui.log")
     ovirt_engine_boot_log = simple_file("/var/log/ovirt-engine/boot.log")
     ovirt_engine_console_log = simple_file("/var/log/ovirt-engine/console.log")
+    ovs_vsctl_list_bridge = simple_command("/usr/bin/ovs-vsctl list bridge")
```
Comment (bfahr) [inline insights/specs/default.py]: @ppandey-redhat  @xiangce this needs a spec for insights_archive as well.

Comment (xiangce) [reply insights/specs/default.py]: Oops, my fault, @ppandey-redhat please raise another PR to add the corresponding `simple_file` for this simple_command defined in the `default.py`

Thank you very much! @bfahr.

Comment (ppandey-redhat) [reply insights/specs/default.py]: @bfahr @xiangce - Acknowledged. As per suggestion, I have raised PR https://github.com/RedHatInsights/insights-core/pull/1774 to add spec for `insights_archive`.

**Pull request merged**